README.md

```diff
@@ -314,6 +316,16 @@ Other optional arguments to `nlsolve`, available for all algorithms, are:
 * `extended_trace`: should additifonal algorithm internals be added to the state
   trace? Default: `false`.
 
+## Fixed Points 
+
+There is a `fixedpoint()` wrapper around `nlsolve()` which maps an input function `F(x)` to `G(x) = F(x) - x`, and likewise for the in-place. This allows convenient solution of fixed-point problems, e.g. of the kind commonly encountered in computational economics. Some notes:
+
+* The default method is `:anderson` with `m = 0`, which corresponds to simple iteration on a contraction mapping.
+* Autodifferentiation is supported; e.g. `fixedpoint(f!, init_x; method = :newtonm. autodiff = :true`).
```

Comment:
I think this should say `... method = :newton, autodiff = :true)` right?
- `:newton` is misspelt `:newtonm` and a full stop is used instead of comma
- ) is outside the back ticks

Reply:
Perfect, thanks. These are just errors.

Reply:
Done.